packages/widgetbook/lib/src/widgetbook_page.dart

```diff
@@ -32,6 +33,9 @@ class WidgetbookPage extends StatelessWidget {
           return AddonInjectorWidget(
             addons: addons,
             routerData: routerData,
+            onChanged: (setting) => context.goTo(
+              queryParams: setting.toQueryParameter(),
+            ),
```

Comment:
very nice!

Reply:
I think we should rename the `goTo` method to something else as it's a bit confusing. The method only changes the given parameters and I think `gotTo` is therefore inadequate as a name. Feel free to do that in another PR.

Reply:
I will handle it in another PR,
#625 is having a different name (`context.queryParameters`) for this, so it will be handled there if this PR landed on `main`.